### DISCUSSION TITLE
Documentation Page has the wrong link

### DISCUSSION THREAD

**Discussion by ostenloo** [vercel/community#1467] [Idea]
In this page https://vercel.com/docs/concepts/deployments/environments the link for "preview environment" is incorrect. It is currently https://vercel.com/docs/concepts/deployments/docs/concepts/deployments/preview-deployments when it should be 
https://vercel.com/docs/concepts/deployments/preview-deployments.

**Comment by amyegan** [comment]
Thanks for reporting this, Austin! I opened a PR to fix it
